planetiler-custommap/src/test/java/com/onthegomap/planetiler/custommap/SchemaYAMLLoadTest.java

```diff
@@ -0,0 +1,35 @@
+package com.onthegomap.planetiler.custommap;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.junit.jupiter.api.Test;
+
+class SchemaYAMLLoadTest {
+
+  /**
+   * Test to ensure that all bundled schemas load to POJOs.
+   * 
+   * @throws Exception
+   */
+  @Test
+  void testSchemaLoad() throws IOException {
+    testSchemasInFolder(Paths.get("src", "main", "resources", "samples"));
+    testSchemasInFolder(Paths.get("src", "test", "resources", "validSchema"));
+  }
+
+  private void testSchemasInFolder(Path path) throws IOException {
+    var schemaFiles = Files.walk(path)
+      .filter(p -> p.getFileName().toString().endsWith(".yml"))
+      .toList();
```

Comment:
maybe also assert the list is not empty just to make sure we're not just failing to find the files.

Reply:
Fixed in f47ed8e